src/lib/Event/ContentCreateFieldOptionsEvent.php

```diff
@@ -11,68 +11,26 @@
 use Ibexa\Contracts\ContentForms\Data\Content\FieldData;
 use Ibexa\Contracts\Core\Repository\Values\Content\ContentCreateStruct;
 use Symfony\Component\Form\FormInterface;
-use Symfony\Contracts\EventDispatcher\Event;
 
-final class ContentCreateFieldOptionsEvent extends Event
+final class ContentCreateFieldOptionsEvent extends StructFieldOptionsEvent
 {
-    /** @var \Ibexa\Contracts\Core\Repository\Values\Content\ContentCreateStruct */
     private $contentCreateStruct;
```

Comment:
You are dropping type here. You probably want to keep either the phpdoc, or straight up declare the type (preferred).